app/assets/stylesheets/donor.css.scss

```diff
@@ -0,0 +1,182 @@
+.donation-cancel {
+  color: $red;
+}
+
+.donation-container {
+  min-height: 0px;
+}
+
+.donation-container, .profile-container {
+  background: $white;
+  font-size: 12px;
+  margin: 0 auto;
+  padding: 15px;
+  width: 68%;
+}
+
+.donation-container b, .profile-container b {
+  font-weight: 600;
+}
+
+.profile-container {
+  min-height: 350px;
+}
+
+.donation-container h1, .profile-container h1 {
+  background: $medium-green;
+  color: $white;
+  display: block;
+  font-size: 18px;
+  font-weight: normal;
+  margin-bottom: 20px;
+  padding: 10px;
+  text-align: center;
+}
+
+.donation-item {
+  background: $donor-profile-light-grey;
+  margin: 10px 0px;
+  padding: 10px;
+  width: 100%;
+}
+
+.donation-item h1 {
+  background: none;
+  color: $medium-green;
+  font-size: 15px;
+  margin: 0px 0px 10px 0px;
+  padding: 0px;
+  text-align: left;
+}
+
+.donation-item-completed {
+  border-left: 10px solid $donation-completed-key;
+}
+
+.donation-item-inprogress {
+  border-left: 10px solid $donation-inprogress-key;
+}
+
+.donation-item-pending {
+  border-left: 10px solid $donation-pending-key;
+}
+
+.donation-key {
+  border-radius: 5px;
+  float: left;
+  padding: 5px 10px;
+  text-align: center;
+}
+
+.donation-key-container {
+  height: 30px;
+  margin-bottom: 20px;
+  width: 100%;
+}
+
+.donation-key-completed {
+  background: $donation-completed-key;
+  color: $white;
+}
+
+.donation-key-description {
+  border-radius: 5px;
+  float: left;
+  margin-right: 20px;
+  padding: 5px 10px;
+}
+
+.donation-key-inprogress {
+  background: $donation-inprogress-key;
+}
+
+.donation-key-pending {
+  background: $donation-pending-key;
+}
+
+.donation-modify {
+  float: right;
+  text-align: right;
+  width: 200px; 
+}
+
+.donation-profile-filler {
+  height: 30px;
+  width: 100%;
+}
+
+.donation-status {
+  float: right;
+  font-weight: 400;
+  text-align: right;
+}
+
+.donation-title {
+  float: left;
+  width: 70%;
+}
+
+.donation-modify-link {
+  float: right;
+}
+
+.profile-img {
+  border: 1px solid $donor-profile-medium-grey;
+  float: left;
+  height: 250px;
+  margin-right: 12px;
+  padding: 8px;
+}
+
+.profile-information {
+  float: left;
+  width: 70%;
+}
+
+.profile-label {
+  color: $dark-green;
+  font-weight: 400;
+}
+
+.profile-summary {
+  background: $donor-profile-donation-summary;
+  float: right;
+  min-height: 240px;
+  padding: 12px;
+  width: 25%;
+}
+
+.profile-summary h2 {
+  color: $medium-green;
+  display: block;
+  font-size: 18px;
+  font-weight: normal;
+  margin-bottom: 25px;
+  margin-top: 10px;
+  text-align: center;
+}
+
+.profile-summary-content {
+  border-radius: 2px;
```

Comment:
Properties should be ordered background, border-radius, float, margin-top, padding